src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas2/OffenderService.kt

```diff
@@ -108,6 +111,24 @@ class OffenderService(
     )
   }
 
+  fun getInfoForPersonOrThrowInternalServerError(crn: String):
+    PersonInfoResult.Success {
+    val personInfo = getInfoForPerson(crn)
```

Comment:
I wonder if we're using `getInfoForPerson` anywhere here - it's quite hard to tell from a quick search. If we're not, we _could_ make it `private` and remove the tests for it, as the behaviour would be tested in the new tests. This could potentially be a refactor on top of this PR though.

Reply:
I have made `getInfoForPerson()` `private` (it's only used in `OffenderService` by the looks of it).  Also removed `GetInfoForPerson` tests since as you say they are superseded by `GetFullInfoForPersonOrThrow` tests now.